cmd/kubeadm/app/apis/kubeadm/v1beta3/doc.go

```diff
@@ -43,6 +43,7 @@ limitations under the License.
 //
 //   - kubeadm v1.15.x and newer can be used to migrate from v1beta1 to v1beta2.
 //   - kubeadm v1.22.x and newer no longer support v1beta1 and older APIs, but can be used to migrate v1beta2 to v1beta3.
+//   - kubeadm v1.27.x and newer no longer support v1beta2 and older APIs.
```

Comment:
but we still don't have something as v1beta4 yet? shouldn't us follow the -2 policy?

Reply:
> shouldn't us follow the -2 policy?

on a number of occasions kubeadm had a singe API version and we haven't followed such a policy.
but there were discussions that the kubeadm-config CM storage version should be the older API version for one release minimum before switching to the newer API.

Reply:
In my mind, the v1beta4 plan is isolated with v1beta2 deprecation.

There is no skew policy for this. https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#version-skew-policy